images/linux/scripts/installers/java-tools.sh

```diff
@@ -29,11 +29,14 @@ echo "JAVA_HOME=/usr/lib/jvm/zulu-${DEFAULT_JDK_VERSION}-azure-amd64" | tee -a /
 echo "JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8" | tee -a /etc/environment
 
 # Install Ant
-apt-fast install -y --no-install-recommends ant ant-optional
-echo "ANT_HOME=/usr/share/ant" | tee -a /etc/environment
+curl -sL https://downloads.apache.org/ant/binaries/apache-ant-1.10.7-bin.zip -o ant.zip
```

Comment:
Does Jmeter require Ant 1.10.7? 
If it is not strictly required, could you please revert Ant installation to `apt`-way? APT package is only slightly behind the official versions (1.10.5 according to https://packages.ubuntu.com/search?suite=bionic&searchon=names&keywords=ant) and we are trying to avoid manual support of versions.

Reply:
Hello @AlenaSviridenko ,
No JMeter does not require Ant.
Regards

Reply:
@AlenaSviridenko in the Java world, we often avoid using these common tools by installing from Linux repositories and rather prefer the binary from the main origin (apache.org).

The issue with the apt-way is the dependencies that it will bring.

I highly recommend we keep it this way.

Reply:
I see, but in my understanding `--no-install-recommends` flag tells Apt to install only the required dependencies. 
Also, changing the way how it is installed now can bring breaking changes on images that can impact someone.